node/src/components/linear_chain_sync/operations.rs

```diff
@@ -139,7 +139,7 @@ where
     let maybe_last_emergency_restart_era_id = chainspec.protocol_config.last_emergency_restart;
 
     let min_era = maybe_last_emergency_restart_era_id.unwrap_or_else(|| EraId::new(0));
-    if min_era >= trusted_header.era_id() {
+    if min_era > trusted_header.era_id() {
```

Comment:
This will be unreachable after we immediately create an switchblock after genesis/restart

Reply:
...could we get rid of this now?

Reply:
And line 133 as well, right?

Reply:
Yup!  Looks good to me...